### PR TITLE
Update message-center.md

### PR DESCRIPTION
**Pull request by msbemba**
Fixed linked and wordings as the Microsoft 365 Service Communications API has now been moved to Microsoft Graph

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/7289

File: microsoft-365/admin/manage/message-center.md [hunk 1]
```diff
@@ -252,7 +252,7 @@ When you open a message in a reading pane, you can use the **Up** and **Down** :
...
 
-For an overview of Message center, see [Message center in Microsoft 365](message-center.md). Or, to learn how to set your language preferences to enable machine translation for Message center posts, see [Language translation for Message center posts](language-translation-for-message-center-posts.md). If you'd like to program an alternative way to get real-time service health information and Message Center communications, please reference [Microsoft 365 Service Communications API Overview](/previous-versions/office/developer/o365-enterprise-developers/jj984343(v=office.15)).
+For an overview of Message center, see [Message center in Microsoft 365](message-center.md). Or, to learn how to set your language preferences to enable machine translation for Message center posts, see [Language translation for Message center posts](language-translation-for-message-center-posts.md). If you'd like to program an alternative way to get real-time service health information and Message Center communications, please reference [Service communications API in Microsoft Graph](/graph/api/resources/service-communications-api-overview?view=graph-rest-beta).
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/manage/message-center.md]:
```suggestion
For an overview of Message center, see [Message center in Microsoft 365](message-center.md). Or, to learn how to set your language preferences to enable machine translation for Message center posts, see [Language translation for Message center posts](language-translation-for-message-center-posts.md). If you'd like to program an alternative way to get real-time service health information and Message center communications, please reference [Working with service communications API in Microsoft Graph](/graph/api/resources/service-communications-api-overview?view=graph-rest-beta).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**